packages/react/script/components-json/build.ts

```diff
@@ -48,8 +54,12 @@ const components = docsFiles.map(docsFilepath => {
   // Example: src/components/Box/Box.docs.json -> src/components/Box/Box.stories.tsx
   const defaultStoryFilepath = docsFilepath.replace(/\.docs\.json$/, '.stories.tsx')
 
+  const isComponentV2 = docs.importPath === '@primer/react/next'
+  const docsName = String(docs.name).toLowerCase()
+  const componentName = isComponentV2 ? `${docsName}v2` : docsName
```

Comment:
I would say it is true for all v2. At least that was my intention. `isComponentV2` won't be true for any of them except Tooltip, that is right but I think this should be expected.

For example select panel v2, since its status is still draft/experimental (assuming the name of the component is `SelectPanel` in the docs), it would use a [prefix](https://github.com/primer/react/blob/main/packages/react/script/components-json/build.ts#L27), in that case it is `drafts` and will generate the storybook id for  `drafts-components-selectpanel--<storyname>`.

When we move SelectPanel v2 to the next bundle, it will go through that `isComponentV2` condition and will be true. Assuming that we will follow the same path for Select Panel v2 as Tooltip v2 (update the status to alpha or beta and move the components out of drafts).

This was my thinking pattern. Let me know if you have any feedback to make it better and more readable!